bin/percy

```diff
@@ -10,24 +10,24 @@ def system!(*args)
 end
 
 FileUtils.chdir APP_ROOT do
+  puts "Enter a 4 digit PR number if available (or just hit enter)"
+  pr_number = gets.chomp
+
   system!({ "PERCY_TOKEN" => Rails.application.credentials.percy_token },
     '
       if [ -z $PERCY_TOKEN ]; then echo "== 🚨 Please download development.key to access PERCY_TOKEN =="; exit 1; fi
 
-      if [ -z "$(git status --porcelain)" ]; then
-        echo "== 📸 Take baseline images =="
-        git checkout main
-        bundle install
-        npx percy exec -- rspec --tag screenshot
-        echo "== 📸 Take new images =="
-        git checkout -
-        bundle install
-        npx percy exec -- rspec --tag screenshot
-      else
-       echo "== 🚨 Please commit changes first =="
-       exit 1
-      fi
-    '
+      if [ -n "$(git status --porcelain)" ]; then echo "== 🚨 Please commit changes first =="; exit 1; fi
```

Comment:
I changed this a bit in that these if statements are gates. If they both pass, then the script runs. Instead of the previous if/else flow.

Reply:
Nice use of `git status --porcelain`! I don't think I knew about it!

```
❯ man git-status | grep -A 4 -F -- --porcelain

       --porcelain[=<version>]
           Give the output in an easy-to-parse format for scripts. This is
           similar to the short output, but will remain stable across Git
           versions and regardless of user configuration. See below for
           details.
```

Reply:
I guess it was here before this pull request, but anyway, it's cool. :)